skipper.go

```diff
@@ -614,6 +614,9 @@ type Options struct {
 	// TLSMinVersion to set the minimal TLS version for all TLS configurations
 	TLSMinVersion uint16
 
+	// List of cipher suites to use for TLS 1.2
```

Comment:
proper godoc starts by the name: `CipherSuites sets the list of cipher suites to use for TLS 1.2`